index.js

```diff
@@ -0,0 +1,135 @@
+document.addEventListener("DOMContentLoaded", () => {
+   setupBtns()
+
+})
+
+const setupBtns = () => {
+   let btnPokemon = document.querySelector("#getPokemon")
+   btnPokemon.addEventListener('click', () => {
+      getPokemons()
+
+   })
+   let btnBattle = document.querySelector("#battlePokemon")
+   btnBattle.addEventListener('click', () => {
+      battlePokemon()
+   })
+}
+
+let getPokemonBtnClicked = false
+
+
+const getPokemons = async () => {
```

Comment:
If you make this function have an argument where the argument is the id of the div, you can reduce the amount of code you have here